apps/masterbots.ai/lib/hooks/use-mb-chat.ts

```
@@ -129,21 +126,30 @@ export function useMBChat(config?: MBChatHookConfig): MBChatHookCallback {
+      } else {
+        aiChatThreadId = activeThread?.threadId
+      }
+
       await Promise.all([
         saveNewMessage({
           role: 'user',
-          threadId:
-            params.threadId || isNewChat ? threadId : activeThread?.threadId,
+          threadId: aiChatThreadId,
           content: userContentRef.current,
           jwt: session!.user?.hasuraJwt
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)